### DISCUSSION TITLE
Raspberry Pi 0 2w support

### DISCUSSION THREAD

**Discussion by RalphyReece** [ptitSeb/box86#966] [Q&A] [answered]
Was wondering if there is a way to run box86 on a 0 2w. I saw that the pi 1 and 0 are not supported, but I have not found any sites saying whether the 2w can.

**Comment by ptitSeb** [comment, marked as answer]
Well, the Interpreter only version is supported on virtually anything that is 32bits little endian. So it would work on 0 2w I suppose. But for the dynarec version (that is the speedy one), you need an arm cpu with Neon support, and I don't think it's supported on the 0 2w...
So, it would probably work, but super slowly... Feel free to try tho.
